eos-tech-support/eos-write-image

```diff
@@ -140,4 +146,42 @@ else
     # Try to format / discard all the sectors on the device
     blkdiscard "$DEVICE" &>/dev/null || true
     cat_image | dd of="$DEVICE" "${DD_OUT_OPTS[@]}"
+
+    # If removable device, delete $ESP/EFI/BOOT/fallback.efi
+    if [ "$REMOVABLE" ]; then
```

Comment:
Can we learn somehow from /sys if a device is removable? People also use eos-write-image with /dev/sdX to make USB sticks etc.